### PR TITLE
Gdpr compliance

### PR DESCRIPTION
**Pull request by AurelReb**
Add GDPR page and inform users about their privacy.
#230

### CODE REVIEW

**Comment by seirl** [comment]
Random idea: maybe all that legalese could go directly in a markdown file? It's just a generic self-contained text with no interaction to the rest of the site.

File: prologin/pages/templates/pages/about-privacy-policy.html [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+            nécessaires au traitement de l'inscription et l'accueil sur site
+            dans de bonnes conditions.</p>
+
+        <h4>Prise de décision automatisée</h4>
+        <p>Le traitement peut inclure une décision entièrement automatisée : les
+            demandes de participation au concours Prologin d'un niveau
+            manifestement insuffisant (évaluation par le juge en ligne Prologin)
+            sont automatiquement écartées. Bien que cette décision automatique
+            ne produise ni effet juridique ni impact significatif, Prologin peut
+            être contacté directement </p>
```
Comment (cyril-amar) [inline prologin/pages/templates/pages/about-privacy-policy.html]: Missing text at the end:
```suggestion
            être <a href="mailto:info@prologin.org">contacté directement</a> pour demander une évaluation manuelle.  </p>
```

**Review by cyril-amar — CHANGES_REQUESTED** [review]

**Comment by cyril-amar** [comment]
> 
> 
> Random idea: maybe all that legalese could go directly in a markdown file? It's just a generic self-contained text with no interaction to the rest of the site.

Why not, it's not like we will need to update it frequently.

**Review by cyril-amar — APPROVED** [review]
Fixed typos in both .po and privacy policy.

**Comment by seirl** [comment]
I removed this from master. Please don't push until you get a review from the site's maintainers.

File: .gitignore [hunk 2]
```diff
@@ -9,6 +9,7 @@ prologin/locale
 
 .idea/
 venv/
+.venv/
```
Comment (seirl) [inline .gitignore:12]: This shouldn't be in this diff.

File: prologin/pages/templates/pages/about-privacy-policy.html [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+{% breadcrumb_for "pages:about-privacy-policy" %}{% trans "Privacy policy" %}
+{% endbreadcrumb_for %}
+{% endblock breadcrumbs %}
+
+{% block title %}{% trans "Privacy policy" %}{% endblock %}
+
+{% block content %}
+
+<h1>{% trans "Privacy policy" %}</h1>
+{% blocktrans %}
```
Comment (seirl) [inline prologin/pages/templates/pages/about-privacy-policy.html:16]: Don't have so much HTML in a blocktrans.

Use `{% blocktrans asvar ...` with markdown inside, then use a markdown filter.

File: prologin/users/templates/users/edit.html [hunk 4]
```diff
@@ -22,7 +22,13 @@ <h1>{% blocktrans with edited_user.username as username %}Editing {{ username }}
 <form method="post" enctype="multipart/form-data">
   {% csrf_token %}
   {% crispy form %}
+  <small>Je consens au traitement de mes données personnelles ci-dessus par Prologin.</small>
```
Comment (seirl) [inline prologin/users/templates/users/edit.html:25]: French?

File: prologin/users/templates/users/edit.html [hunk 5]
```diff
@@ -22,7 +22,13 @@ <h1>{% blocktrans with edited_user.username as username %}Editing {{ username }}
 <form method="post" enctype="multipart/form-data">
   {% csrf_token %}
   {% crispy form %}
+  <small>Je consens au traitement de mes données personnelles ci-dessus par Prologin.</small> 
   <button class="btn btn-primary btn-block" type="submit"><i class="fa fa-save"></i> {% trans "Save" %}</button>
+    <p class="help-block ">
+      Prologin est organisés par l'association Prologin, qui agit en tant que contrôleur de données. Les réponses collectées dans ce formulaire nous permettent de fournir les services (communication et juge en ligne) et préparer le concours : sélections, restrictions alimentaires, etc...
```
Comment (seirl) [inline prologin/users/templates/users/edit.html:28]: French?

File: prologin/users/templates/users/register.html [hunk 6]
```diff
@@ -18,11 +18,21 @@ <h1>{% trans "Register" %}</h1>
 <form method="POST" class="form-horizontal" action="{% url 'users:register' %}?{% querystring request next=next %}">
   {% csrf_token %}
   {{ form|bootstrap_horizontal }}
+  <div class="form-group">
+    <small class="col-sm-offset-2 col-sm-10">Je consens au traitement de mes données personnelles ci-dessus par Prologin.</small>
```
Comment (seirl) [inline prologin/users/templates/users/register.html:22]: French?

File: prologin/users/templates/users/register.html [hunk 7]
```diff
@@ -18,11 +18,21 @@ <h1>{% trans "Register" %}</h1>
...
+    <small class="col-sm-offset-2 col-sm-10">Je consens au traitement de mes données personnelles ci-dessus par Prologin.</small> 
+  </div>
   <div class="form-group">
     <div class="col-sm-offset-2 col-sm-10">
       <input class="btn btn-primary" type="submit" value="{% trans "Register" %}">
     </div>
   </div>
+  <div class="form-group">
+    <p class="help-block col-sm-offset-2 col-sm-10">
+      Prologin est organisés par l'association Prologin, qui agit en tant que contrôleur de données. Les réponses collectées dans ce formulaire nous permettent de fournir les services (communication et juge en ligne) et préparer le concours : sélections, restrictions alimentaires, etc...
```
Comment (seirl) [inline prologin/users/templates/users/register.html:31]: French?

**Pull request merged**